SudokuBF/SudokuBF.cpp

```diff
@@ -235,8 +235,15 @@ int bruteForce(std::string_view strField)
 
 int main(int argc, char *argv[])
 {
-	if (argc != 2)
+	if (argc != 2) {
+		std::cerr<<"Single parameter required: "<<unit_size<<"x"<<unit_size<<" sudoku puzzle as first argument (single string with "<<field_size<<" symbols)."<<std::endl;
```

Comment:
Let's use more canonical form of passive voice:
parameter is required
vs
required